src/vm/ecalllist.h

```diff
@@ -540,6 +540,7 @@ FCFuncEnd()
 FCFuncStart(gAssemblyFuncs)
     QCFuncElement("GetEntryAssemblyNative", AssemblyNative::GetEntryAssembly)
     QCFuncElement("GetExecutingAssemblyNative", AssemblyNative::GetExecutingAssembly)
+    FCFuncElement("GetAssemblyCount", AssemblyNative::GetAssemblyCount)
```

Comment:
You got `FEATURE_EVENT_TRACE` around some of the places, but not others. I do think you need it to have it under ifdef at all.